astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -2209,7 +2228,8 @@ def ancestors(
         if context is None:
             context = InferenceContext()
         if not self.bases and self.qname() != "builtins.object":
-            yield builtin_lookup("object")[1][0]
+            # This should always be a ClassDef (which we don't assert for)
+            yield builtin_lookup("object")[1][0]  # type: ignore[misc]
```

Comment:
I'd be okay with an assertion, but it's not a must-do either.

Reply:
We use this so often I would be more inclined to create a `get_builtin_classdef` but for now I think we should just move on. There a probably much better solutions when we have a better codebase.